src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -71,12 +72,39 @@ class NetworkOPsImp final
     : public NetworkOPs
     , public beast::DeadlineTimer::Listener
 {
-public:
-    enum Fault
+    /**
+     * Transaction with input flags and results to be applied in batches.
+     */
+    class TransactionStatus
+    {
+    public:
+        Transaction::pointer transaction;
```

Comment:
struct usually implies POD. Less important for this particular piece of code. Personally, I would have used struct. And no one should be thinking in terms of C. Ever.